src/components/Dashboard/Link.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import React, { Component, Fragment } from 'react';
+import { bindActionCreators } from 'redux';
+
+import { actions as dashboardActions } from 'reducers/dashboard';
+import { getDashboardComponent } from 'selectors/dashboard';
+
+export class Link extends Component {
+  static propTypes = {
+    actions: PropTypes.object.isRequired,
+    children: PropTypes.any.isRequired,
+    name: PropTypes.string,
+    to: PropTypes.string.isRequired,
+    item: PropTypes.any,
+    className: PropTypes.string
+  };
+
+  constructor(props) {
+    super(props);
+    this.select = this.select.bind(this);
+  }
+
+  select(name, item = null) {
+    const { actions } = this.props;
+
+    actions.selectDashboard({
+      name,
+      item
+    });
+  }
+
+  render() {
+    const { to, name, item, className } = this.props;
+
+    return (
+      <Fragment>
+        <a
+          href={to}
+          onClick={e => this.select(name, item, e)}
```

Comment:
`e` is passed as a third argument that is not present in the `select` method signature. I think `() =>this.select(name, item)` works.

Reply:
fixed